modules/gcp_helpers/authorization/main.tf

```diff
@@ -0,0 +1,36 @@
+resource "google_service_account" "ocean_integration_service_account" {
+  project      = var.project
+  account_id   = var.service_account_id
+  display_name = "Ocean Integration Service Account"
+}
+
+resource "google_organization_iam_custom_role" "ocean_integration_iam_org_role" {
+  title       = "The Ocean Integration organization role"
+  role_id     = var.role_id
+  permissions = var.permissions
+  org_id      = var.organization
+}
+
+resource "google_service_account_iam_binding" "ocean_integration_iam_org_binding" {
+  service_account_id = google_service_account.ocean_integration_service_account.name
```

Comment:
I don't know it it's intentional but just making sure it's correct 
service_acount_**id** = service_account.**name**

Reply:
![image](https://github.com/port-labs/terraform-ocean-integration-factory/assets/51418643/14330830-03d3-478f-9904-2b31b08a7044)
Yup, seems like bad naming on their side